cns/middlewares/k8sSwiftV2.go

```diff
@@ -128,6 +128,12 @@ func (m *K8sSWIFTv2Middleware) validateIPConfigsRequest(ctx context.Context, req
 		if mtpnc.Status.PrimaryIP == "" || mtpnc.Status.MacAddress == "" || mtpnc.Status.NCID == "" || mtpnc.Status.GatewayIP == "" {
 			return nil, types.UnexpectedError, errMTPNCNotReady.Error()
 		}
+		interfaceInfos := mtpnc.Status.InterfaceInfos
+		for _, interfaceInfo := range interfaceInfos {
+			if interfaceInfo.NICType == string(cns.BackendNIC) {
+				req.BackendInterfaceMacAddress = interfaceInfo.MacAddress
```

Comment:
looks like this should be done outside of validate request func

Reply:
CNS read MTPNC value within that function, it's being used today to set SecondaryInterfacesExist flag, similar to that, instead of setting boolean value, I am setting macaddress.